frontend/public/components/_nav.scss

```diff
@@ -52,39 +53,39 @@ $active-border: 3px;
   -webkit-overflow-scrolling: touch;
 }
 
+.navigation-container--open {
+  .navigation-container__list {
+    margin: 5px 0 10px 0;
+  }
+  .navigation-container__item__separator {
```

Comment:
BEM says don't nest elements.  This should be `.navigation-container__separator`.  Here and below.  I realize you're following the existing code, but we should probably clean them all up.